### PR TITLE
Temporal types with order by

### PR DESCRIPTION
**Pull request by antonio2368**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Future
+
+### Major Features and Improvements
+
+### Bug Fixes
+
+* Allow for temporal types to be used with `ORDER BY` clause. [#299](https://github.com/memgraph/memgraph/pull/299)
```
Comment (antaljanosbenjamin) [inline docs/changelog.md:13]: I might be too pedantic, so feel free to ignore my comment. "Allow" means more likely "adding a feature" and not fixing the bug. If I am not mistaken, memgraph just crashed in this case, so I would rephrase this:
```suggestion
* Fix crash when temporal types are used with `ORDER BY` clause. [#299](https://github.com/memgraph/memgraph/pull/299)
```

**Review by jbajic — APPROVED** [review]

**Review by antaljanosbenjamin — APPROVED** [review]
